src/app/not-found.tsx

```diff
@@ -0,0 +1,10 @@
+const Custom404 = ({ message }: { message: string }) => {
+  return (
+    <>
```

Comment:
what does this look like? Just because it's a 404 page, doesn't mean we should drop our standards for visual polish

Reply:
<img width="1435" alt="Screenshot 2024-04-29 at 11 26 57 AM" src="https://github.com/Teamable-Analytics/teamable-frontend/assets/77403071/9d3b809a-9e5a-43b7-bea0-1072cd328ae3">
It looks like this now, let me know what you think